There is a CI failure on the CI, the "Other checks" check of the build/normal configuration. I don't know what is going wrong. The log around the failure reads like this:

```
rm -f config.log config.status libtool
++ git status --porcelain
+ test -z ''
++ git ls-files --others -i --exclude-standard
+ test -z 'tools/ocamlobjinfo
tools/ocamlobjinfo.opt'
Error: Process completed with exit code 1.
```

This looks related to the following lines from the Checks of tools/ci/actions/runner.sh:

```
  # check that the `distclean` target definitely cleans the tree
  $MAKE distclean
  # Check the working tree is clean
  test -z "$(git status --porcelain)"
  # Check that there are no ignored files
  test -z "$(git ls-files --others -i --exclude-standard)"
```

(I wish the CI would echo these comments instead of having them as comments, so that we can have a sense of what it is doing by reading the CI logs.)

--- a/Makefile
+++ b/Makefile
@@ -529,7 +529,8 @@ endif
 # the configuration is not available during clean so we don't
 # know whether they have been configured / built or not
 clean::
-	rm -f $(addprefix tools/ocamlopt,p p.opt p.exe p.opt.exe)
+	rm -f $(addprefix tools/ocamlopt,p p.opt p.exe p.opt.exe) \
+	  tools/ocamlobjinfo $(addprefix tools/ocamlobjinfo,.opt .exe .opt.exe)
 
 TOOLS_NAT = $(TOOLS_TO_INSTALL_NAT)
 TOOLS_BYT = $(TOOLS_TO_INSTALL_BYT) dumpobj primreq stripdebug cmpbyt

